cmd/search.go

```diff
@@ -18,17 +18,29 @@ import (
 	"errors"
 	"fmt"
 	"os"
+	"strings"
 
 	"github.com/dropbox/dropbox-sdk-go-unofficial/files"
 	"github.com/spf13/cobra"
 )
 
 func search(cmd *cobra.Command, args []string) (err error) {
-	if len(args) != 1 {
+	if len(args) == 0 {
 		return errors.New("`search` requires a `query` argument")
 	}
 
-	arg := files.NewSearchArg("", args[0])
+	// Parse path scope, if provided.
+	scope := ""
+	if len(args) == 2 {
+		scope = args[1]
+		if len(scope) > 0 {
```

Comment:
I'm actually not sure how we'd get `len(args) == 2` with `len(scope) == 0`. Even if we did, the prefix check below will still fail for it. So just remove the outer if?